### DISCUSSION TITLE
Some submarines can't be stealthily infiltrated by monsters who can open doors

### DISCUSSION THREAD

**Discussion by TheNinjaScout** [FakeFishGames/Barotrauma#15095] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Some submarines are for some reason not boardable by enemies with ID cards such as husks, forcing them to break the airlock door instead of just using the buttons to enter the sub quickly:

Barsuk (While outer airlock is closed instead of inner)
Berilia (While outer airlock is closed instead of inner)
Camel
Humpback (Can be boarded but all inner normal doors inside can't be opened by monsters not on the same team as the crew, intended?)
Remora

### Reproduction steps

1. Use one of the submarines listed in the description of the issue
2. Spawn husks or any monster that can open doors
3. Notice how they will have massive trouble getting in without breaking the airlocks even though they have access to the airlock buttons

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Rokvach** [comment]
Fixed and tested the fix to be working correctly. Fix coming in the next update. Closing discussion.
